src/libstd/lib.rs

```diff
@@ -422,6 +423,11 @@ pub use core_collections::vec;
 #[stable(feature = "rust1", since = "1.0.0")]
 pub use std_unicode::char;
 
+// Reexport the start module on platforms that provide it
+#[unstable(feature = "start_fn", issue="0")]
+#[cfg(target_os = "redox")]
+pub use sys::start::*;
```

Comment:
The thing is that it must be exposed at the crate root. I could have pub use sys::start; in libstd::rt, but then rt::start::* would still have to be pub used in lib.rs